studyzen/Utils/UserActionStamp.cs

```diff
@@ -0,0 +1,12 @@
+namespace StudyZen.Utils;
+
+public record UserActionStamp
+{
+    public string User { get; init; }
+    public DateTime Timestamp { get; init; }
+    public UserActionStamp(string user = "anonymous")
+    {
+        User = user;
+        Timestamp = DateTime.Now;
```

Comment:
I think it would be better to use `DateTime.UtcNow`, since that would allow adjusting for timezone offsets. However not sure how necessary it is at this stage since all consumers are from Lithuania :D

Reply:
![image](https://github.com/vu-code-cartel/studyzen/assets/80688723/4d7fab54-cd71-4325-aa76-977dcdc29a75)

short form

Reply:
fixed